.github/workflows/release.yml

```diff
@@ -1,34 +1,38 @@
-name: GitHub Release of nRF Connect for Desktop
+name: Release to GitHub
 
 on:
-  workflow_dispatch:
-    inputs:
-      tag:
-        description: 'Tag to be used (e.g. v2.1.3)'
-        required: true
-        type: string
-      prerelease:
-        description: 'Make the release a pre-release'
-        required: true
-        default: true
-        type: boolean
-      draft:
-        description: 'Create draft release'
-        required: true
-        default: true
-        type: boolean
+    workflow_dispatch:
+        inputs:
+            run_id:
+                description: 'The workflow id where the artifacts were uploaded'
+                required: true
+                type: string
+            tag:
+                description: 'Tag to be used (e.g. v8.8.8)'
+                required: true
+                type: string
 
 jobs:
-  release:
-    runs-on: ['ubuntu-latest']
-    steps:
-      - name: Create Release
-        uses: ncipollo/release-action@v1.13.0
-        with:
-            allowUpdates: false
-            artifacts: 'release/*'
-            commit: ${{ inputs.tag }}
-            prerelease: ${{ inputs.prerelease }}
-            draft: ${{ inputs.draft }}
+    release:
+        runs-on: ['ubuntu-latest']
+        steps:
+            - name: Download artifacts
+              uses: dawidd6/action-download-artifact@v3
+              with:
+                  run_id: ${{ inputs.run_id }}
```

Comment:
Shouldn't it actually be very easy to replace this with a command line using `gh`?

We could run 

```
gh run download ${{ inputs.run_id }} --repo nordicsemiconductor/pc-nrfconnect-launcher
```